### ISSUE TITLE
Control Center is missing keyboard navigation hinting

### ISSUE THREAD

**Issue by flexiondotorg** [flexiondotorg/ubuntu-mate-themes#53]
While attempting to navigate MATE Control Center using just the keyboard, there is not visible hinting to show which items are currently selected. This issue is present in Ambiant-MATE, Radiant-MATE and Ambiant-MATE-Dark.

**Comment by flexiondotorg** [comment]
I was wrong, this is working correctly.
